tests/ballerina-unit-test/src/test/resources/test-src/io/client_socket_io.bal

```diff
@@ -84,6 +84,7 @@ function readRecord(io:Socket socket) returns string[]|error {
             }
         }
     } else {
-        return { message: "No records found" };
+        error err = error("No records found");
+        return err;
```

Comment:
err variable is redundant

Reply:
You meant this "return error("No records found");"? It doesn't work.